translations/Hindi/special-terms.md

```diff
@@ -0,0 +1,107 @@
+ like unification, and since it is used in Zulip, it would mean unifying 
+ Zulip with something. , फेसबुक के साथ जुलिप एकीकरण
+* Notification : अधिसूचना , अधिसूचना is the easiest to understand translation 
+of "Notification" and it literally means "Before information/Pre-information",
+ which is basically what notifications are: Alerting the user about a piece of
+ information waiting for them. , अधिसूचनाएं बंद करो
+* Alert Word : सतर्क शब्द , चेतावनी,सावधान,भय which are translations of alert 
+are all related to being scared and as a result alert. Therefore सतर्क is used
+ as a translation of alert. , अपने सतर्क शब्द देखें
+* View : देखें , One translation of view is राय, but that means opinion. 
+Therefore the formal translation of see is used here , बिना पढ़े हुए पोस्ट देखें
+* Home : मुख्य पृष्ठ , Translation of homepage is used here with a slight 
+modification. मुख पृष्ठ(home page) literally means face page or front page,
+ usually associated with books and magazines. Therefore मुख्य पृष्ठ is used
+ which means "Main page" , मुख्य पृष्ठ पर जाएं
+* Emoji : इमोजी , Transliteration used here, as there are no possible 
+translations for "emoji". Also the common Hindi folk is familiar with the 
+English word and therefore using transliteration shouldn't be a problem , इमोजी
+ शैली बदलें
+## Phrases (वाक्यांश)
+* Subscribe to a stream/Unsubscribe from a stream : किसी स्ट्रीम की सदस्यता लें
+ / एक स्ट्रीम से सदस्यता समाप्त करें , There is no direct word for "Subscribe"
+ in Hindi, but the translator result means "Take a subscription". Similarly the
+ result of "Unsubscribe" means "End a subscription". Both of these translations
+ are used here , <stream> की सदस्यता लें / सदस्यता रद्द करें
+* Narrow to : द्वारा संकुचित करें , The translation of "Narrow by" is used 
+instead of "Narrow to", which when translated means "physically narrow an 
+object to something", which cannot be used in this context , इस विषय द्वारा 
+संकुचित करें
+* Filter : फ़िल्टर , One available translation of filter is छन्ना, but it 
+literally means a strainer. Therefore translitration is used here. , फ़िल्टर 
+हटाए
+* Mute/Unmute : म्यूट / अनम्यूट , Transliteration is used here as welll as 
+translation of mute will give results which mean mute, but in the context as 
+a person who cannot speak. , इस <stream> को म्यूट / अनम्यूट करें
+* Deactivate/Reactivate : निष्क्रिय करें / पुन: सक्रिय करें , These two words 
+are direct translations and there is no alternative translation to either of 
+them which we can use in Zulip as they are long phrases , अपने खाते को 
+निष्क्रिय / पुनः सक्रिय करें
+* Search : खोज , खोज is used in many search engines as well, and is a well 
+known word. Other translations like तलाश and जाँच are used in the context of 
+security, where people are searched, therefore cannot be used here. , आपने <
+search item> के लिए खोजा
+* Pin : पिन , Transliteration used here as well for two reasons: Translations 

```

Comment:
Pinning is similar to saving, so perhaps we can look at how other services translate save in a computing sense and use that.

Reply:
Will try to find out and update here

Reply:
Cross-checked in Whatsapp where we can pin the chats, it's the same translation